### PR TITLE
[PLAT-767] an admin can browse quiz questions

### PR DESCRIPTION
**Pull request by preziu**
- [ ] index quiz questions on production

### CODE REVIEW

File: resources/assets/js/admin/router.js [hunk 1]
```diff
@@ -58,19 +58,17 @@ let routes = [
...
-				path: 'edit/:quizId',
-				component: require('js/admin/components/quizes/QuizQuestionEdit.vue')
-			},
-			{
-				name: 'quiz-creator',
-				path: 'new',
-				component: require('js/admin/components/quizes/QuizQuestionCreate.vue')
-			}
-		],
+		component: require('js/admin/components/quizes/list/QuizQuestionsList.vue'),
```
Comment (kvas-damian) [inline resources/assets/js/admin/router.js]: we don't use `require` in `router.js`. You can find the new approach on master branch

File: resources/assets/js/admin/components/quizes/list/QuizQuestionListItem.vue [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+				name: 'quiz-editor',
+				params: {
+					quizId: this.id,
+				},
+			};
+		},
+	},
+	methods: {
+		stripHtml(text) {
+			var tmp = document.createElement('div');
```
Comment (gitkuba) [inline resources/assets/js/admin/components/quizes/list/QuizQuestionListItem.vue]: If this method is to stay though, we need to `let` that `var` go. 😆

File: resources/assets/js/admin/components/quizes/list/QuizQuestionListItem.vue [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+			return {
+				name: 'quiz-editor',
+				params: {
+					quizId: this.id,
+				},
+			};
+		},
+	},
+	methods: {
+		stripHtml(text) {
```
[the remark this answers is not in the document]
Comment (preziu) [reply resources/assets/js/admin/components/quizes/list/QuizQuestionListItem.vue]: Displaying questions with html tags makes them take too much space. 
Imo admins do not need to see formatted snippets of these questions - they only need the essence of it such as text and id number. The full and formatted question snippet can be found in edit view.

File: resources/assets/js/admin/components/quizes/list/QuizQuestionListItem.vue (hunk 2 above)
Comment (preziu) [reply resources/assets/js/admin/components/quizes/list/QuizQuestionListItem.vue]: 🙈

File: resources/assets/js/admin/components/quizes/list/QuizQuestionListItem.vue [hunk 4]
```diff
@@ -22,7 +22,7 @@ export default {
 	},
 	methods: {
 		stripHtml(text) {
-			var tmp = document.createElement('div');
+			let tmp = document.createElement('div');
```
Comment (kvas-damian) [inline resources/assets/js/admin/components/quizes/list/QuizQuestionListItem.vue]: you can use const here

Comment (preziu) [reply resources/assets/js/admin/components/quizes/list/QuizQuestionListItem.vue]: changed 🎉

File: resources/assets/js/admin/components/quizes/list/QuizQuestionsList.vue [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+</template>
+
+<script>
+import WnlQuizQuestionListItem from 'js/admin/components/quizes/list/QuizQuestionListItem';
+import WnlPaginatedList from 'js/admin/components/lists/PaginatedList';
+
+export default {
+	name: 'QuizQuestionsList',
+	components: {
+		'quiz-question-list-item': WnlQuizQuestionListItem,
```
Comment (bognix) [inline resources/assets/js/admin/components/quizes/list/QuizQuestionsList.vue]: use the same patent as with the WnlPaginatedList.

File: resources/assets/js/admin/components/quizes/list/QuizQuestionListItem.vue [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+		<router-link :to="to" class="link">
+			<span class="id">{{id}}. {{stripHtml(text)}}</span>
+		</router-link>
+	</div>
+</template>
+
+<script>
+export default {
+	name: 'QuizQuestionListItem',
+	props: ['id', 'text'],
```
Comment (bognix) [inline resources/assets/js/admin/components/quizes/list/QuizQuestionListItem.vue]: create props validation.

File: resources/assets/js/admin/components/quizes/list/QuizQuestionsList.vue [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+</template>
+
+<script>
+import WnlQuizQuestionListItem from 'js/admin/components/quizes/list/QuizQuestionListItem';
+import WnlPaginatedList from 'js/admin/components/lists/PaginatedList';
+
+export default {
+	name: 'QuizQuestionsList',
+	components: {
+		'wnl-quiz-question-list-item': WnlQuizQuestionListItem,
```
Comment (kvas-damian) [inline resources/assets/js/admin/components/quizes/list/QuizQuestionsList.vue]: you can remove `'wnl-quiz-question-list-item':`

**Pull request merged**